lib/ini/ini.nit

```diff
@@ -235,7 +237,7 @@ class ConfigTree
 		key = key.substring(0, key.length - 2)
 		var len = 0
 		if has_key(key) then
-			len = get_node(key).children.length
+			len = get_node(key).as(not null).children.length
```

Comment:
`self[key]` seems to already return a nullable String. You may use it instead of `as(not null)`

Reply:
I need a ConfigNode and not a String?

Reply:
then use `get_node` if you need a node :p

~~~nit
var node = get_node(key)
if node != null then len = node.children.length
~~~